test/worlds/particle_emitter.sdf

```diff
@@ -0,0 +1,120 @@
+          <anchor line="top" target="window" target_line="top"/>
+          <anchor line="bottom" target="window" target_line="bottom"/>
+        </ignition-gui>
+
+        <engine>ogre</engine>
+        <scene>scene</scene>
+        <ambient_light>0.4 0.4 0.4</ambient_light>
+        <background_color>0.8 0.8 0.8</background_color>
+        <camera_pose>-1 0 1 0 0.5 0</camera_pose>
+      </plugin>
+
+      <!-- World control -->
+      <plugin filename="WorldControl" name="World control">
+        <ignition-gui>
+          <title>World control</title>
+          <property type="bool" key="showTitleBar">false</property>
+          <property type="bool" key="resizable">false</property>
+          <property type="double" key="height">72</property>
+          <property type="double" key="width">121</property>
+          <property type="double" key="z">1</property>
+          <anchor line="left" target="window" target_line="left"/>
+          <anchor line="bottom" target="window" target_line="bottom"/>
+        </ignition-gui>
+
+        <play_pause>true</play_pause>
+        <step>true</step>
+        <start_paused>true</start_paused>
+
+      </plugin>
+    </gui>
+
+    <light type="directional" name="sun">
+      <cast_shadows>true</cast_shadows>
+      <pose>0 0 10 0 0 0</pose>
+      <diffuse>1 1 1 1</diffuse>
+      <specular>0.5 0.5 0.5 1</specular>
+      <attenuation>
+        <range>1000</range>
+        <constant>0.9</constant>
+        <linear>0.01</linear>
+        <quadratic>0.001</quadratic>
+      </attenuation>
+      <direction>-0.5 0.1 -0.9</direction>
+    </light>
+
+    <model name="ground_plane">
+      <static>true</static>
+      <link name="link">
+        <collision name="collision">
+          <geometry>
+            <plane>
+              <normal>0 0 1</normal>
+            </plane>
+          </geometry>
+        </collision>
+        <visual name="visual">
+          <geometry>
+            <plane>
+              <normal>0 0 1</normal>
+              <size>100 100</size>
+            </plane>
+          </geometry>
+          <material>
+            <ambient>0.8 0.8 0.8 1</ambient>
+            <diffuse>0.8 0.8 0.8 1</diffuse>
+            <specular>0.8 0.8 0.8 1</specular>
+          </material>
+        </visual>
+      </link>
+    </model>
+
+    <model name="smoke_generator_demo_model">
+      <pose>0 0 0 0 0 0</pose>
+      <static>true</static>
+      <link name="link">
+        <visual name="visual">
+          <geometry>
+            <box>
+              <size>1 1 1</size>
+            </box>
+          </geometry>
+        </visual>
+      </link>
+
+      <plugin filename="ignition-gazebo-particle-emitter-system"
+        name="ignition::gazebo::systems::ParticleEmitter">
+        <emitter_name>smoke_emitter</emitter_name>
+        <emitting>true</emitting>
+        <lifetime>2</lifetime>
+        <min_velocity>10</min_velocity>
+        <max_velocity>20</max_velocity>
+        <scale_rate>10</scale_rate>
```

Comment:
It would be nice to add another test case with invalid values for these 4 parameters (along with `duration`) to make sure that we handle bad input correctly.

Reply:
If we were to add another test case, perhaps we can include the other parameters that are missing from this test case, such as `color_start` and `color_end`.

Reply:
https://github.com/ignitionrobotics/ign-gazebo/pull/562/commits/9ed110b9105131a8c82ae86013bd8fdb4264fc21